packages/blockchain-wallet/src/FabricWallet.ts

```diff
@@ -52,6 +52,41 @@ export class FabricWallet implements IFabricWallet {
         return  this.wallet.list();
     }
 
+    public async getIDs(identities: string[]): Promise<FabricIdentity[]> {
+        const allIDs: FabricIdentity[] = [];
```

Comment:
this won't work with v2, see the `getIdentities` function for what it should look like